udata_front/theme/gouvfr/assets/js/components/DataservicesSearch/DataservicesSearch.vue

```diff
@@ -0,0 +1,213 @@
+        <div v-if="dataservices !== null && dataservices.data.length">
+          <ul class="fr-mt-1w border-default-grey border-top relative z-2">
+            <li v-for="dataservice in dataservices.data" :key="dataservice.id">
+              <DataserviceCard :dataservice :dataserviceUrl="dataservice.self_web_url" />
+            </li>
+          </ul>
+          <Pagination
+              v-if="dataservices !== null && dataservices.total > dataservices.page_size"
+              :page="page"
+              :pageSize="dataservices.page_size"
+              :totalResults="dataservices.total"
+              @change="changePage"
+              class="fr-mt-2w"
+            />
+        </div>
+        <div v-if="dataservices !== null && dataservices.data.length === 0" class="fr-mt-2w">
+          <ActionCard
+          :title="t('No result found for your search')"
+          :icon="franceWithMagnifyingGlassIcon"
+          type="primary"
+          >
+            <p class="fr-mt-1v fr-mb-3v">
+              {{ t("Try to reset filters to widen your search.") }}<br/>
+              {{ t("You can also give us more details with our feedback form.") }}
+            </p>
+            <template v-slot:actions>
+              <button @click="resetFilters" class="fr-btn fr-btn--secondary">
+                {{ t("Reset filters") }}
+              </button>
+              <a :href="data_search_feedback_form_url" class="fr-btn fr-btn--tertiary-no-outline fr-btn--icon-left fr-icon-lightbulb-line fr-ml-1w">
+                {{ t("Tell us what you are looking for") }}
+              </a>
+            </template>
+          </ActionCard>
+        </div>
+      </section>
+    </div>
+  </form>
+</template>
+
+<script setup lang="ts">
+import { Dataservice, DataserviceCard, Pagination } from "@datagouv/components/ts";
+import { ref, onMounted, computed, useId, useTemplateRef, watchEffect, watch } from "vue";
+import { useI18n } from 'vue-i18n';
+import Loader from "../dataset/loader.vue";
+import MultiSelect from "../MultiSelect/MultiSelect.vue";
+import ActionCard from "../Form/ActionCard/ActionCard.vue";
+import { data_search_feedback_form_url } from "../../config";
+import { api } from "../../plugins/api";
+import franceWithMagnifyingGlassIcon from "../../../../templates/svg/illustrations/france_with_magnifying_glass.svg";
+import { PaginatedArray } from "../../api/types";
+import { refDebounced } from '@vueuse/core'
+
+const { t } = useI18n();
+
+const organization = ref<string | null>(null);
+const isRestricted = ref<boolean | null>(null);
+const isRestrictedId = useId();
+
+const page = ref(1);
+
+const hasFilters = computed(() => {
+  return organization.value || isRestricted.value !== null
+});
+const resetFilters = () => {
+  organization.value = '';
+  isRestricted.value = null;
+}
+
+const searchId = useId();
+const searchQuery = ref('');
+const searchQueryDebounced = refDebounced(searchQuery, 500);
+const searchInput = useTemplateRef('searchInput');
```

Comment:
Can you rename this or update vue to 3.5.6 ? There is a [bug](https://github.com/vuejs/core/issues/11795#issuecomment-2326858438) that cause a warning because the variable name is the same as the ref

Reply:
Done in 92e2b8ac